src/o365/yammer/commands.ts

```diff
@@ -7,6 +7,7 @@ export default {
   YAMMER_MESSAGE_LIST: `${prefix} message list`,
   YAMMER_MESSAGE_REMOVE: `${prefix} message remove`,
   YAMMER_NETWORK_LIST: `${prefix} network list`,
+  YAMMER_REPORT_GROUPACTIVITYCOUNTS: `${prefix} report groupactivityfilecounts`,
```

Comment:
Renamed from `groupactivityfilecounts` to `groupsactivitycounts` to match the issue description.